django/utils/http.py

```diff
@@ -33,6 +33,12 @@
 RFC3986_GENDELIMS = str(":/?#[]@")
 RFC3986_SUBDELIMS = str("!$&'()*+,;=")
 
+PROTOCOL_TO_PORT = {
+    'http': 80,
+    'https': 443,
+    'ftp': 21,
```

Comment:
The only protocols serving a Django site should be http and https indeed. For the sake of performance, should we still use `socket.getservbyname()` or can we stick with the hard-coded dict? I mean, how often have the http and https ports changed [in the time since Django exists]?